app/components/latest-releases-card.hbs

```diff
@@ -0,0 +1,51 @@
+<div class="bg-white dark:bg-gray-850 rounded-md border shadow-sm dark:border-white/10 p-5" data-test-latest-releases>
+  <h4 class="font-semibold text-gray-700 dark:text-gray-200 mb-3" data-test-latest-releases-title>
```

Comment:
```suggestion
  <h4 class="font-semibold text-gray-700 dark:text-gray-200 mb-3">
```

Pretty much pointless asserting this - it's just a static string. In tests we usually only assert things that have some logic to them (like a dynamic page title, or formatted date etc.)